webview_panels/.storybook/__mocks__/vscode.ts

```
@@ -5,14 +5,25 @@ let func = undefined;
 export const vscode = {
   postMessage: async (args: Record<string, unknown>) => {
     if (args.syncRequestId) {
-      const body = typeof func === 'function' ? (func as Function)(args) : data;
+      let response;
+      let status = true;
+      try {
+        const body =
+          typeof func === "function" ? await (func as Function)(args) : data;
```

Biome

[error] Don't use 'Function' as a type.

Prefer explicitly define the function shape. This type accepts any function-like value, which can be a common source of bugs.

(lint/complexity/noBannedTypes)